applications/luci-app-aria2/Makefile

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2016 Openwrt.org
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI Support for Aria2
+LUCI_DEPENDS:=+aria2 +!PACKAGE_webui-aria2:yaaw
```

Comment:
webui_aria2 is a little big, I am wondering whether to add it.

In fact luci-aria2 does not depend on webui-ariaw nor yaaw.